.github/workflows/publish-test-result-from-fork.yml

```diff
@@ -16,46 +16,9 @@ jobs:
          github.event.sender.login == 'dependabot[bot]' ||
          github.event.workflow_run.head_repository.full_name != github.repository
       )
-    steps:
-      - name: Download Artifacts
-        uses: actions/github-script@v3
-        with:
-          script: |
-            var fs = require('fs');
-            var path = require('path');
-            var artifacts_path = path.join('${{github.workspace}}', 'artifacts')
-            fs.mkdirSync(artifacts_path, { recursive: true })
-            var artifacts = await github.actions.listWorkflowRunArtifacts({
-               owner: context.repo.owner,
-               repo: context.repo.repo,
-               run_id: ${{ github.event.workflow_run.id }},
-            });
-            for (const artifact of artifacts.data.artifacts) {
-               var download = await github.actions.downloadArtifact({
-                  owner: context.repo.owner,
-                  repo: context.repo.repo,
-                  artifact_id: artifact.id,
-                  archive_format: 'zip',
-               });
-               var artifact_path = path.join(artifacts_path, `${artifact.name}.zip`)
-               fs.writeFileSync(artifact_path, Buffer.from(download.data));
-               console.log(`Downloaded ${artifact_path}`);
-            }
-      - name: Extract Artifacts
-        run: |
-          for file in artifacts/*.zip
-          do
-            if [ -f "$file" ]
-            then
-              dir="${file/%.zip/}"
-              mkdir -p "$dir"
-              unzip -d "$dir" "$file"
-            fi
-          done
-      - name: Publish Linux UT Results
-        uses: EnricoMi/publish-unit-test-result-action@v1
-        with:
-          commit: ${{ github.event.workflow_run.head_sha }}
-          files: artifacts/linux-ut-result-*/**/*.xml
-          check_name: Linux Test Report
-          comment_title: Linux Test Report
+    uses: ./.github/workflows/publish-test-result-from-fork-action.yml
```

Comment:
Can you give me some tips? Although I googled the relevant materials, I was still not sure how to solve this problem:

```
Error: Can't find 'action.yml', 'action.yaml' or 'Dockerfile' under '/home/runner/work/OpenMLDB/OpenMLDB/.github/workflows/publish-test-results-action.yml'. Did you forget to run actions/checkout before running your local action?
```

Reply:
I think use checkout action should fix this

Reply:
Yes, the PR is mostly fine for me. I'll try merge it this week, but before that fix the `uses` patch problem.